### PR TITLE
Cardinal updates

### LINKED ISSUE #182: Update Cardinal models to latest NekRS version
We recently updated the NekRS version used in Cardinal. This version of NekRS is not backwards compatible with the previous version, and requires some updates to input file syntax.

### PR DESCRIPTION
**Pull request by aprilnovak**
This PR updates syntax in the Cardinal cases to be compatible with the newer NekRS version (which we have updated in Cardinal). I also made some minor changes to the 67 pebble bed Cardinal case, to use `Cardinal` in place of `CARDINAL` (for consistency elsewhere) and added a bit more context/info where I thought it could help improve clarity.

@hapfang I am not sure what is the new syntax for the `preconditioner = semg_amg`. It doesn't exist any more in v22 - do you happen to know what I should change that to?

Closes #182

### CODE REVIEW

**Review by GiudGiud — COMMENTED** [review]
thanks for the fixes. 
I ll wait until the preconditioner discussion happens.

I think we should update the Cardinal submodule in apps/cardinal too?

**Comment by aprilnovak** [comment]
Good point, I wasn't sure if the submodules lived in this repo or not. I can update

File: htgr/pb67_cardinal/pb67.par [hunk 1]
```diff
@@ -23,7 +19,7 @@ filterModes = 2
 [PRESSURE]
 residualTol = 1e-04
 residualProj = yes
-preconditioner = semg
+#preconditioner = semg
```
[the remark this answers is not in the document]
Comment (GiudGiud) [reply htgr/pb67_cardinal/pb67.par:22]: @hapfang @aprilnovak waiting for resolution on this

[the remark this answers is not in the document]
Comment (aprilnovak) [reply htgr/pb67_cardinal/pb67.par:22]: Perhaps @emerzari knows?

[the remark this answers is not in the document]
Comment (emerzari) [reply htgr/pb67_cardinal/pb67.par:22]: Please check the release notes of v22.  I think all that needs to be done should be to replace that line with:

coarseGridDiscretization = SEMFEM

Comment (GiudGiud) [inline htgr/pb67_cardinal/pb67.par:22]:
```suggestion
coarseGridDiscretization = SEMFEM
```

**Review by GiudGiud — APPROVED** [review]

**Comment by aprilnovak** [comment]
Sorry @GiudGiud for the delays on this - before you merge, we are testing this input file to be sure that the preconditioner change still retains good performance (otherwise it's possible the solve just aborts early-on due to high CFL).

@jma6166 have you or David (can't find his github username) have tested this yet?

**Comment by GiudGiud** [comment]
Sounds good please let me know.
i ll add another commit for that openmc input failure

**Comment by aprilnovak** [comment]
Thanks! I know how to fix that error, so feel free to assign it to me.

**Comment by GiudGiud** [comment]
Sounds good I ll let you do it then

**Comment by GiudGiud** [comment]
Got it on the other PR it s fine

**Comment by jma6166** [comment]
@aprilnovak we haven’t tested it fully, will update you tomorrow.

**Comment by GiudGiud** [comment]
You dont need the submodule update anymore, I did that already

**Comment by GiudGiud** [comment]
should do it

**Review by GiudGiud — APPROVED** [review]
Thanks for the update!

**Pull request merged**